vips/vip-171/vip-171-testnet.ts

```diff
@@ -0,0 +1,129 @@
+import { ethers } from "hardhat";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const NEW_VBEP20_DELEGATE_IMPL = "0xAC5CFaC96871f35f7ce4eD2b46484Db34B548b40";
+const ACCESS_CONTROL_MANAGER = "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA";
+const NORMAL_TIMELOCK = "0xce10739590001705F7FF231611ba4A48B2820327";
+const TREASURY = "0x8b293600c50d6fbdc6ed4251cc75ece29880276f";
+
+interface AssetConfig {
+  name: string;
+  address: string;
+  reduceReservesBlockDelta: number;
+  acceptAdmin: boolean;
+}
+
+export const CORE_MARKETS: AssetConfig[] = [
```

Comment:
Why are not considering every VToken deployed to testnet? (https://docs-v4.venus.io/deployed-contracts/core-pool#vtokens-1)

Reply:
Added more markets [b14dbd0](https://github.com/VenusProtocol/vips/pull/67/commits/b14dbd0f1bdbf039afc0def2690aa63f99018b55)